experiment/invitations-accepter/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/flagutil"
+	"k8s.io/test-infra/prow/github"
+)
+
+type options struct {
+	tokenPaths flagutil.Strings
+
+	dryRun bool
+}
+
+type githubClient interface {
+	ListCurrentUserRepoInvitations() ([]github.UserRepoInvitation, error)
+	AcceptUserRepoInvitation(invitationID int) error
+	BotUser() (*github.UserData, error)
+}
+
+func gatherOptions() options {
+	o := options{}
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
Please use the stadard flagutil.GitHub options, ppl can just call the tool multiple times if they want to use it for multiple accounts

Reply:
updated.